src/main/java/teammates/common/datatransfer/FeedbackResponseCommentAttributes.java

```diff
@@ -191,9 +191,7 @@ public String getJsonString() {
     @Override
     public void sanitizeForSaving() {
         this.commentText = Sanitizer.sanitizeTextField(this.commentText);
```

Comment:
`sanitizeTextField` removes leading and trailing white spaces only. While `sanitizeForRichText` uses html sensitization, so we can't decommission one of them. But for this case we can refactor it and add the call to `sanitizeTextField` within `sanitizeForRichText`.